jme3-vehicle-creator/src/com/jme3/gde/vehiclecreator/VehicleEditorController.java

```diff
@@ -77,10 +77,13 @@ public VehicleEditorController(JmeSpatial jmeRootNode, BinaryModelDataObject cur
         result.addLookupListener(this);
         toolsNode.addLight(new DirectionalLight());
         Node track = (Node) new DesktopAssetManager(true).loadModel("Models/Racetrack/Raceway.j3o");
-        track.getChild("Grass").getControl(RigidBodyControl.class).setPhysicsLocation(new Vector3f(30, -1, 0));
-        track.getChild("Grass").getControl(RigidBodyControl.class).setPhysicsRotation(new Quaternion().fromAngleAxis(FastMath.HALF_PI * 0.68f, Vector3f.UNIT_Y).toRotationMatrix());
-        track.getChild("Road").getControl(RigidBodyControl.class).setPhysicsLocation(new Vector3f(30, 0, 0));
-        track.getChild("Road").getControl(RigidBodyControl.class).setPhysicsRotation(new Quaternion().fromAngleAxis(FastMath.HALF_PI * 0.68f, Vector3f.UNIT_Y).toRotationMatrix());
+        if(track.getChild("Grass").getControl(RigidBodyControl.class) == null){
+            track.getChild("Grass").addControl(new RigidBodyControl());
+            track.getChild("Road").addControl(new RigidBodyControl());
```

Comment:
Not sure if we'd want individual if's here? Though that would look ugly and be broken use-cases, I guess.
If we know that the Raceway.j3o doesn't have any RBCs at all anymore (which is very likely, considering we don't have them in blender and probably it would need a gltf extension anyway), we can probably also remove the if clause.

Reply:
But I've added them manually to the j3o. So in that case the whole clause could be removed. Or should I remove them from the j3o and generate every time?

Reply:
Oh. I mean if they'd be dynamically generated, then we'd probably get rid of the problem that caused #278.
Otherwise, we may have another breakage when/if physics is changing again.

I guess both ways are fine for now though, especially since you already had the work to add it as well.
Do you see any downsides with code adding them though? any tweeks?

Reply:
If we're to keep the lines generating the RBC's (and keep the j3o with the RBC's) I think it would also be better to do a check first. Or how would it handle generating a new control if there already is one?